src/components/OverlappingCard/OverlappingCard.tsx

```diff
@@ -1,15 +1,51 @@
 import * as React from 'react';
 import style from './OverlappingCard.st.css';
-import { Card, CardProps } from '../Card';
+import { CardRatioOptions } from '../Card';
 
-export interface OverlappingCardProps extends CardProps {}
+export { CardRatioOptions as OverlappingCardRatioOptions };
 
-export const OverlappingCard = (props: OverlappingCardProps) => {
-  const { ratio, flippedRatio, invertInfoPosition } = props;
+export interface OverlappingCardProps {
+  media?: React.ReactNode;
+  info?: React.ReactNode;
+  ratio?: CardRatioOptions;
+  flippedRatio?: boolean;
+  invertInfoPosition?: boolean;
+}
+
+const OverlappingCard = ({
+  info,
+  media,
+  ratio,
+  invertInfoPosition,
+  flippedRatio,
+  ...rest
+}: OverlappingCardProps) => {
   return (
-    <Card
-      {...props}
-      {...style('root', { ratio, flippedRatio, invertInfoPosition }, props)}
-    />
+    <div
+      {...style(
+        'root',
+        {
+          ratio: media ? ratio : CardRatioOptions.RATIO_100,
```

Comment:
is this logic tested?

Reply:
@shlomitc yes test 'should render ratio 100 if there is no media' line 59 the test was already written when Overlapping was part of Card